openapi3.yaml

```diff
@@ -29,8 +29,6 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/jobResponse'
-        204:
```

Comment:
if that was actually implemented removing it is breaking change

Reply:
I didnt say it wrong.
I said its breaking and you should create tasks for fixing the broken client